web/src/views/Apply/FormQuestions.tsx

```diff
@@ -0,0 +1,190 @@
+import Image from "next/image";
+import deleteIcon from "~public/icons/close.svg";
+import addIcon from "~public/icons/plus.svg";
+import {Form, Formik, Field, ErrorMessage, FieldArray} from "formik";
+import * as Yup from 'yup';
+
+const styles = {
+  label: 'block text-gray-700 text-sm font-bold pt-2 pb-1',
+  field:
+    'bg-gray-200 text-gray-700 focus:outline-none focus:shadow-outline border border-gray-300 rounded py-2 px-4 block w-full appearance-none',
+  dynamicField:
+    'bg-gray-200 text-gray-700 focus:outline-none focus:shadow-outline border border-gray-300 rounded py-2 px-4 block w-full appearance-none',
+  button:
+    ' bg-gray-700 text-white font-bold py-2 px-4 w-full rounded hover:bg-gray-600',
+  errorMsg: 'text-red-500 text-sm',
+}
+
+export const FormQuestions = () => {
+  const formSchema = Yup.object().shape({
+    firstName: Yup.string().required('Campo obligatorio'),
+    lastName: Yup.string().required('Campo obligatorio'),
+    email: Yup.string().email('Correo inválido').required('Campo obligatorio'),
+    institution:  Yup.string().required('Campo obligatorio'),
+    links: Yup.array()
+      .of(
+        Yup.object().shape({
+          url: Yup.string().required('Campo obligatorio'),
+        })
+      )
+      .required('Tiene que colocar al menos 1 link'),
```

Comment:
We could add a limit to the number of links. 5 links sound enough to me. What do you think?

Reply:
sure